services/server/test/helpers/assertions.ts

```diff
@@ -29,7 +29,7 @@ export const assertValidationError = (
 
 // If you pass storageService = false, then the match will not be compared to the database
 export const assertVerification = async (
-  storageService: StorageService | null,
+  sourcifyDatabase: SourcifyDatabaseService | null,
```

Comment:
Why are we switching to the DB in all of this file? Isn't the whole point abstracting away the Storage?

Reply:
This is temporary, it is going to be an sql pool later, I prepared the tests for that. Didn't want to include also this in current PR.